test/verify/check-journal

```diff
@@ -468,9 +476,13 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
 
     @skipImage("ABRT not available", "debian-stable", "debian-testing", "ubuntu-stable",
                "ubuntu-2004", "fedora-coreos", "rhel-8-3", "rhel-8-3-distropkg", "centos-8-stream")
+    @nondestructive
     def testAbrtSegv(self):
         b = self.browser
+        m = self.machine
 
+        m.execute("systemctl restart abrt-journal-core")
+        self.addCleanup(m.execute, "rm -rf /var/spool/abrt/*")
```

Comment:
At some point it makes sense to split out a TestAbrt class, with this as common setUp(). This would also avoid the duplication of the skipImages. This can be a follow-up, of course.

But let's use restore_dir() instead for consistency?

Reply:
Yeah, I though the same that it could be split up, but as a follow up.

> But let's use restore_dir() instead for consistency?

Sure, I can do that, but there is not much to restore, we really just need to clean up after ourselves.

Reply:
OK, thanks - fine to keep then.